great_expectations/data_context/data_context/file_data_context.py

```diff
@@ -98,7 +99,12 @@ def root_directory(self) -> Optional[str]:
         return self._context_root_directory
 
     def _init_variables(self) -> FileDataContextVariables:
-        raise NotImplementedError
+        variables: FileDataContextVariables = FileDataContextVariables(
+            config=self._project_config,
+            substitutions={},
```

Comment:
nit: For consistency since this is set to {} on init if not provided can you remove this line?